catalogue/webapp/test/integration/stacks.test.js

```diff
@@ -2,13 +2,14 @@ const authUrl = 'https://id.wellcomecollection.org/oauth2';
 
 xdescribe('logged out', () => {
   beforeAll(async () => {
+    jest.setTimeout(30000);
```

Comment:
You willing to let a test run for 5 minutes before telling you it's broken?

Reply:
30 seconds? Long enough to start the app.